eisbuk-admin/src/components/atoms/AttendanceCard/AttendanceCard.tsx

```diff
@@ -17,17 +19,19 @@ const AttendanceCard: React.FC<Props> = ({
   userBookings,
   absentees,
 }) => {
+  const dispatch = useDispatch();
   // convert timestamp to luxon for easier processing
-  const luxonStart = fb2Luxon(date);
 
+  const luxonStart = fb2Luxon(date);
   // convert durations to number values
+
   const durationNumbers = durations.map((duration) => Number(duration));
   const longestDuration = Math.max(...durationNumbers);
-
   // get end time of longest duration (we're still using durations here, so it's pretty straightforward)
-  const luxonEnd = luxonStart.plus({ minutes: longestDuration });
 
+  const luxonEnd = luxonStart.plus({ minutes: longestDuration });
   // get time for rendering
```

Comment:
comments should be on top of line/first line of a block they're refering to. 
Like this:
```typescript
// below is a 'name' variable
const name = "Walter"

// below is a 'surname' variable
const surname = "White"
```
Not like this:

```typescript
// below is a 'name' variable

const name = "Walter"
// below is a 'surname' variable

const surname = "White"
```